from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(878, 622)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.verticalLayoutWidget = QtWidgets.QWidget(self.centralwidget)
        self.verticalLayoutWidget.setGeometry(QtCore.QRect(10, 10, 221, 361))
        self.verticalLayoutWidget.setObjectName("verticalLayoutWidget")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.verticalLayoutWidget)
        self.verticalLayout.setContentsMargins(0, 0, 0, 0)
        self.verticalLayout.setObjectName("verticalLayout")
        self.pushButton = QtWidgets.QPushButton(self.verticalLayoutWidget)
        self.pushButton.setEnabled(True)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(11)
        font.setBold(False)
        font.setItalic(False)
        font.setUnderline(False)
        font.setWeight(50)
        font.setStrikeOut(False)
        font.setKerning(True)
        self.pushButton.setFont(font)
        self.pushButton.setObjectName("pushButton")
        self.verticalLayout.addWidget(self.pushButton)
        self.pushButton_2 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(11)
        font.setBold(False)
        font.setItalic(False)
        font.setUnderline(False)
        font.setWeight(50)
        font.setStrikeOut(False)
        font.setKerning(True)
        self.pushButton_2.setFont(font)
        self.pushButton_2.setObjectName("pushButton_2")
        self.verticalLayout.addWidget(self.pushButton_2)
        self.pushButton_6 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(11)
        font.setBold(False)
        font.setItalic(False)
        font.setUnderline(False)
        font.setWeight(50)
        font.setStrikeOut(False)
        font.setKerning(True)
        self.pushButton_6.setFont(font)
        self.pushButton_6.setObjectName("pushButton_6")
        self.verticalLayout.addWidget(self.pushButton_6)
        self.pushButton_4 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(11)
        font.setBold(False)
        font.setItalic(False)
        font.setUnderline(False)
        font.setWeight(50)
        font.setStrikeOut(False)
        font.setKerning(True)
        self.pushButton_4.setFont(font)
        self.pushButton_4.setObjectName("pushButton_4")
        self.verticalLayout.addWidget(self.pushButton_4)
        self.pushButton_5 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(11)
        font.setBold(False)
        font.setItalic(False)
        font.setUnderline(False)
        font.setWeight(50)
        font.setStrikeOut(False)
        font.setKerning(True)
        self.pushButton_5.setFont(font)
        self.pushButton_5.setObjectName("pushButton_5")
        self.verticalLayout.addWidget(self.pushButton_5)
        self.pushButton_3 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(11)
        font.setBold(False)
        font.setItalic(False)
        font.setUnderline(False)
        font.setWeight(50)
        font.setStrikeOut(False)
        font.setKerning(True)
        self.pushButton_3.setFont(font)
        self.pushButton_3.setObjectName("pushButton_3")
        self.verticalLayout.addWidget(self.pushButton_3)
        self.pushButton_7 = QtWidgets.QPushButton(self.verticalLayoutWidget)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(11)
        font.setBold(False)
        font.setItalic(False)
        font.setUnderline(False)
        font.setWeight(50)
        font.setStrikeOut(False)
        font.setKerning(True)
        self.pushButton_7.setFont(font)
        self.pushButton_7.setObjectName("pushButton_7")
        self.verticalLayout.addWidget(self.pushButton_7)
        self.stackedwitged = QtWidgets.QStackedWidget(self.centralwidget)
        self.stackedwitged.setGeometry(QtCore.QRect(320, 70, 361, 221))
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(9)
        font.setBold(False)
        font.setWeight(50)
        self.stackedwitged.setFont(font)
        self.stackedwitged.setObjectName("stackedwitged")
        self.page_8 = QtWidgets.QWidget()
        self.page_8.setObjectName("page_8")
        self.stackedwitged.addWidget(self.page_8)
        self.page = QtWidgets.QWidget()
        self.page.setObjectName("page")
        self.verticalLayoutWidget_2 = QtWidgets.QWidget(self.page)
        self.verticalLayoutWidget_2.setGeometry(QtCore.QRect(30, 20, 291, 181))
        self.verticalLayoutWidget_2.setObjectName("verticalLayoutWidget_2")
        self.clients = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_2)
        self.clients.setContentsMargins(0, 0, 0, 0)
        self.clients.setObjectName("clients")
        self.pushButton_8 = QtWidgets.QPushButton(self.verticalLayoutWidget_2)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(9)
        font.setBold(False)
        font.setWeight(50)
        self.pushButton_8.setFont(font)
        self.pushButton_8.setObjectName("pushButton_8")
        self.clients.addWidget(self.pushButton_8)
        self.pushButton_9 = QtWidgets.QPushButton(self.verticalLayoutWidget_2)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(9)
        font.setBold(False)
        font.setWeight(50)
        self.pushButton_9.setFont(font)
        self.pushButton_9.setObjectName("pushButton_9")
        self.clients.addWidget(self.pushButton_9)
        self.pushButton_10 = QtWidgets.QPushButton(self.verticalLayoutWidget_2)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(9)
        font.setBold(False)
        font.setWeight(50)
        self.pushButton_10.setFont(font)
        self.pushButton_10.setObjectName("pushButton_10")
        self.clients.addWidget(self.pushButton_10)
        self.stackedwitged.addWidget(self.page)
        self.page_2 = QtWidgets.QWidget()
        self.page_2.setObjectName("page_2")
        self.verticalLayoutWidget_3 = QtWidgets.QWidget(self.page_2)
        self.verticalLayoutWidget_3.setGeometry(QtCore.QRect(30, 20, 291, 181))
        self.verticalLayoutWidget_3.setObjectName("verticalLayoutWidget_3")
        self.accounts = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_3)
        self.accounts.setContentsMargins(0, 0, 0, 0)
        self.accounts.setObjectName("accounts")
        self.pushButton_12 = QtWidgets.QPushButton(self.verticalLayoutWidget_3)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(9)
        font.setBold(False)
        font.setWeight(50)
        self.pushButton_12.setFont(font)
        self.pushButton_12.setObjectName("pushButton_12")
        self.accounts.addWidget(self.pushButton_12)
        self.pushButton_11 = QtWidgets.QPushButton(self.verticalLayoutWidget_3)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(9)
        font.setBold(False)
        font.setWeight(50)
        self.pushButton_11.setFont(font)
        self.pushButton_11.setObjectName("pushButton_11")
        self.accounts.addWidget(self.pushButton_11)
        self.pushButton_13 = QtWidgets.QPushButton(self.verticalLayoutWidget_3)
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(9)
        font.setBold(False)
        font.setWeight(50)
        self.pushButton_13.setFont(font)
        self.pushButton_13.setObjectName("pushButton_13")
        self.accounts.addWidget(self.pushButton_13)
        self.stackedwitged.addWidget(self.page_2)
        self.page_3 = QtWidgets.QWidget()
        self.page_3.setObjectName("page_3")
        self.verticalLayoutWidget_4 = QtWidgets.QWidget(self.page_3)
        self.verticalLayoutWidget_4.setGeometry(QtCore.QRect(30, 20, 291, 121))
        self.verticalLayoutWidget_4.setObjectName("verticalLayoutWidget_4")
        self.transactions = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_4)
        self.transactions.setContentsMargins(0, 0, 0, 0)
        self.transactions.setObjectName("transactions")
        self.pushButton_14 = QtWidgets.QPushButton(self.verticalLayoutWidget_4)
        self.pushButton_14.setObjectName("pushButton_14")
        self.transactions.addWidget(self.pushButton_14)
        self.pushButton_16 = QtWidgets.QPushButton(self.verticalLayoutWidget_4)
        self.pushButton_16.setObjectName("pushButton_16")
        self.transactions.addWidget(self.pushButton_16)
        self.stackedwitged.addWidget(self.page_3)
        self.page_4 = QtWidgets.QWidget()
        self.page_4.setObjectName("page_4")
        self.verticalLayoutWidget_5 = QtWidgets.QWidget(self.page_4)
        self.verticalLayoutWidget_5.setGeometry(QtCore.QRect(30, 20, 291, 181))
        self.verticalLayoutWidget_5.setObjectName("verticalLayoutWidget_5")
        self.credits = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_5)
        self.credits.setContentsMargins(0, 0, 0, 0)
        self.credits.setObjectName("credits")
        self.pushButton_15 = QtWidgets.QPushButton(self.verticalLayoutWidget_5)
        self.pushButton_15.setObjectName("pushButton_15")
        self.credits.addWidget(self.pushButton_15)
        self.pushButton_17 = QtWidgets.QPushButton(self.verticalLayoutWidget_5)
        self.pushButton_17.setObjectName("pushButton_17")
        self.credits.addWidget(self.pushButton_17)
        self.pushButton_18 = QtWidgets.QPushButton(self.verticalLayoutWidget_5)
        self.pushButton_18.setObjectName("pushButton_18")
        self.credits.addWidget(self.pushButton_18)
        self.stackedwitged.addWidget(self.page_4)
        self.page_5 = QtWidgets.QWidget()
        self.page_5.setObjectName("page_5")
        self.verticalLayoutWidget_6 = QtWidgets.QWidget(self.page_5)
        self.verticalLayoutWidget_6.setGeometry(QtCore.QRect(30, 20, 291, 181))
        self.verticalLayoutWidget_6.setObjectName("verticalLayoutWidget_6")
        self.cards = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_6)
        self.cards.setContentsMargins(0, 0, 0, 0)
        self.cards.setObjectName("cards")
        self.pushButton_19 = QtWidgets.QPushButton(self.verticalLayoutWidget_6)
        self.pushButton_19.setObjectName("pushButton_19")
        self.cards.addWidget(self.pushButton_19)
        self.pushButton_20 = QtWidgets.QPushButton(self.verticalLayoutWidget_6)
        self.pushButton_20.setObjectName("pushButton_20")
        self.cards.addWidget(self.pushButton_20)
        self.pushButton_21 = QtWidgets.QPushButton(self.verticalLayoutWidget_6)
        self.pushButton_21.setObjectName("pushButton_21")
        self.cards.addWidget(self.pushButton_21)
        self.stackedwitged.addWidget(self.page_5)
        self.page_6 = QtWidgets.QWidget()
        self.page_6.setObjectName("page_6")
        self.verticalLayoutWidget_7 = QtWidgets.QWidget(self.page_6)
        self.verticalLayoutWidget_7.setGeometry(QtCore.QRect(20, 20, 321, 181))
        self.verticalLayoutWidget_7.setObjectName("verticalLayoutWidget_7")
        self.employees = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_7)
        self.employees.setContentsMargins(0, 0, 0, 0)
        self.employees.setObjectName("employees")
        self.pushButton_22 = QtWidgets.QPushButton(self.verticalLayoutWidget_7)
        self.pushButton_22.setObjectName("pushButton_22")
        self.employees.addWidget(self.pushButton_22)
        self.pushButton_23 = QtWidgets.QPushButton(self.verticalLayoutWidget_7)
        self.pushButton_23.setObjectName("pushButton_23")
        self.employees.addWidget(self.pushButton_23)
        self.pushButton_24 = QtWidgets.QPushButton(self.verticalLayoutWidget_7)
        self.pushButton_24.setObjectName("pushButton_24")
        self.employees.addWidget(self.pushButton_24)
        self.pushButton_25 = QtWidgets.QPushButton(self.verticalLayoutWidget_7)
        self.pushButton_25.setObjectName("pushButton_25")
        self.employees.addWidget(self.pushButton_25)
        self.stackedwitged.addWidget(self.page_6)
        self.page_7 = QtWidgets.QWidget()
        self.page_7.setObjectName("page_7")
        self.verticalLayoutWidget_8 = QtWidgets.QWidget(self.page_7)
        self.verticalLayoutWidget_8.setGeometry(QtCore.QRect(30, 20, 291, 71))
        self.verticalLayoutWidget_8.setObjectName("verticalLayoutWidget_8")
        self.activitylog = QtWidgets.QVBoxLayout(self.verticalLayoutWidget_8)
        self.activitylog.setContentsMargins(0, 0, 0, 0)
        self.activitylog.setObjectName("activitylog")
        self.pushButton_29 = QtWidgets.QPushButton(self.verticalLayoutWidget_8)
        self.pushButton_29.setObjectName("pushButton_29")
        self.activitylog.addWidget(self.pushButton_29)
        self.stackedwitged.addWidget(self.page_7)
        self.tabwidget = QtWidgets.QTabWidget(self.centralwidget)
        self.tabwidget.setEnabled(True)
        self.tabwidget.setGeometry(QtCore.QRect(110, 380, 751, 201))
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(8)
        font.setBold(False)
        font.setWeight(50)
        self.tabwidget.setFont(font)
        self.tabwidget.setObjectName("tabwidget")
        self.tab = QtWidgets.QWidget()
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setPointSize(6)
        self.tab.setFont(font)
        self.tab.setObjectName("tab")
        self.clientstableview = QtWidgets.QTableView(self.tab)
        self.clientstableview.setGeometry(QtCore.QRect(0, 10, 461, 141))
        self.clientstableview.setObjectName("clientstableview")
        self.clientscombobox = QtWidgets.QComboBox(self.tab)
        self.clientscombobox.setGeometry(QtCore.QRect(470, 10, 161, 31))
        self.clientscombobox.setObjectName("clientscombobox")
        self.pushButton_27 = QtWidgets.QPushButton(self.tab)
        self.pushButton_27.setGeometry(QtCore.QRect(470, 50, 161, 31))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.pushButton_27.setFont(font)
        self.pushButton_27.setObjectName("pushButton_27")
        self.tabwidget.addTab(self.tab, "")
        self.tab_2 = QtWidgets.QWidget()
        self.tab_2.setObjectName("tab_2")
        self.accountstableview = QtWidgets.QTableView(self.tab_2)
        self.accountstableview.setGeometry(QtCore.QRect(0, 10, 461, 141))
        self.accountstableview.setObjectName("accountstableview")
        self.accountscombobox = QtWidgets.QComboBox(self.tab_2)
        self.accountscombobox.setGeometry(QtCore.QRect(470, 10, 161, 31))
        self.accountscombobox.setObjectName("accountscombobox")
        self.pushButton_28 = QtWidgets.QPushButton(self.tab_2)
        self.pushButton_28.setGeometry(QtCore.QRect(470, 50, 161, 31))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.pushButton_28.setFont(font)
        self.pushButton_28.setObjectName("pushButton_28")
        self.tabwidget.addTab(self.tab_2, "")
        self.tab_3 = QtWidgets.QWidget()
        self.tab_3.setObjectName("tab_3")
        self.transactionstableview = QtWidgets.QTableView(self.tab_3)
        self.transactionstableview.setGeometry(QtCore.QRect(0, 10, 461, 141))
        self.transactionstableview.setObjectName("transactionstableview")
        self.transactionscombobox = QtWidgets.QComboBox(self.tab_3)
        self.transactionscombobox.setGeometry(QtCore.QRect(470, 10, 161, 31))
        self.transactionscombobox.setObjectName("transactionscombobox")
        self.transactionsdateEditStart = QtWidgets.QDateEdit(self.tab_3)
        self.transactionsdateEditStart.setGeometry(QtCore.QRect(470, 50, 161, 31))
        self.transactionsdateEditStart.setObjectName("transactionsdateEditStart")
        self.transactionsdateEditEnd = QtWidgets.QDateEdit(self.tab_3)
        self.transactionsdateEditEnd.setGeometry(QtCore.QRect(470, 90, 161, 31))
        self.transactionsdateEditEnd.setObjectName("transactionsdateEditEnd")
        self.pushButton_30 = QtWidgets.QPushButton(self.tab_3)
        self.pushButton_30.setGeometry(QtCore.QRect(470, 130, 161, 31))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.pushButton_30.setFont(font)
        self.pushButton_30.setObjectName("pushButton_30")
        self.tabwidget.addTab(self.tab_3, "")
        self.tab_7 = QtWidgets.QWidget()
        self.tab_7.setObjectName("tab_7")
        self.creditstableview = QtWidgets.QTableView(self.tab_7)
        self.creditstableview.setGeometry(QtCore.QRect(0, 10, 461, 141))
        self.creditstableview.setObjectName("creditstableview")
        self.creditscombobox = QtWidgets.QComboBox(self.tab_7)
        self.creditscombobox.setGeometry(QtCore.QRect(470, 10, 161, 31))
        self.creditscombobox.setObjectName("creditscombobox")
        self.pushButton_31 = QtWidgets.QPushButton(self.tab_7)
        self.pushButton_31.setGeometry(QtCore.QRect(470, 50, 161, 31))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.pushButton_31.setFont(font)
        self.pushButton_31.setObjectName("pushButton_31")
        self.tabwidget.addTab(self.tab_7, "")
        self.tab_6 = QtWidgets.QWidget()
        self.tab_6.setObjectName("tab_6")
        self.cardstableview = QtWidgets.QTableView(self.tab_6)
        self.cardstableview.setGeometry(QtCore.QRect(0, 10, 461, 141))
        self.cardstableview.setObjectName("cardstableview")
        self.cardscombobox = QtWidgets.QComboBox(self.tab_6)
        self.cardscombobox.setGeometry(QtCore.QRect(470, 10, 161, 31))
        self.cardscombobox.setObjectName("cardscombobox")
        self.pushButton_32 = QtWidgets.QPushButton(self.tab_6)
        self.pushButton_32.setGeometry(QtCore.QRect(470, 50, 161, 31))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.pushButton_32.setFont(font)
        self.pushButton_32.setObjectName("pushButton_32")
        self.tabwidget.addTab(self.tab_6, "")
        self.tab_5 = QtWidgets.QWidget()
        self.tab_5.setObjectName("tab_5")
        self.employeestableview = QtWidgets.QTableView(self.tab_5)
        self.employeestableview.setGeometry(QtCore.QRect(0, 10, 461, 141))
        self.employeestableview.setObjectName("employeestableview")
        self.empoloyeescombobox = QtWidgets.QComboBox(self.tab_5)
        self.empoloyeescombobox.setGeometry(QtCore.QRect(470, 10, 161, 31))
        self.empoloyeescombobox.setObjectName("empoloyeescombobox")
        self.pushButton_33 = QtWidgets.QPushButton(self.tab_5)
        self.pushButton_33.setGeometry(QtCore.QRect(470, 50, 161, 31))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.pushButton_33.setFont(font)
        self.pushButton_33.setObjectName("pushButton_33")
        self.tabwidget.addTab(self.tab_5, "")
        self.tab_4 = QtWidgets.QWidget()
        self.tab_4.setObjectName("tab_4")
        self.activitylogtableview = QtWidgets.QTableView(self.tab_4)
        self.activitylogtableview.setGeometry(QtCore.QRect(0, 10, 461, 141))
        self.activitylogtableview.setObjectName("activitylogtableview")
        self.activitylogcombobox = QtWidgets.QComboBox(self.tab_4)
        self.activitylogcombobox.setGeometry(QtCore.QRect(470, 10, 161, 31))
        self.activitylogcombobox.setObjectName("activitylogcombobox")
        self.activitylogsdateEditStart = QtWidgets.QDateEdit(self.tab_4)
        self.activitylogsdateEditStart.setGeometry(QtCore.QRect(470, 50, 161, 31))
        self.activitylogsdateEditStart.setObjectName("activitylogsdateEditStart")
        self.activitylogsdateEditEnd = QtWidgets.QDateEdit(self.tab_4)
        self.activitylogsdateEditEnd.setGeometry(QtCore.QRect(470, 90, 161, 31))
        self.activitylogsdateEditEnd.setObjectName("activitylogsdateEditEnd")
        self.pushButton_34 = QtWidgets.QPushButton(self.tab_4)
        self.pushButton_34.setGeometry(QtCore.QRect(470, 130, 161, 31))
        font = QtGui.QFont()
        font.setPointSize(8)
        self.pushButton_34.setFont(font)
        self.pushButton_34.setObjectName("pushButton_34")
        self.tabwidget.addTab(self.tab_4, "")
        self.pushButton_26 = QtWidgets.QPushButton(self.centralwidget)
        self.pushButton_26.setGeometry(QtCore.QRect(700, 10, 151, 31))
        font = QtGui.QFont()
        font.setFamily("Sitka Small")
        font.setBold(False)
        font.setWeight(50)
        self.pushButton_26.setFont(font)
        self.pushButton_26.setObjectName("pushButton_26")
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 878, 26))
        self.menubar.setObjectName("menubar")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)

        self.retranslateUi(MainWindow)
        self.stackedwitged.setCurrentIndex(0)
        self.tabwidget.setCurrentIndex(6)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.pushButton.setText(_translate("MainWindow", "Клиенты"))
        self.pushButton_2.setText(_translate("MainWindow", "Счета"))
        self.pushButton_6.setText(_translate("MainWindow", "Транзакции"))
        self.pushButton_4.setText(_translate("MainWindow", "Кредиты"))
        self.pushButton_5.setText(_translate("MainWindow", "Карты"))
        self.pushButton_3.setText(_translate("MainWindow", "Сотрудники"))
        self.pushButton_7.setText(_translate("MainWindow", "Журнал активности"))
        self.pushButton_8.setText(_translate("MainWindow", "Добавить клиента"))
        self.pushButton_9.setText(_translate("MainWindow", "Редактировать клиента"))
        self.pushButton_10.setText(_translate("MainWindow", "Удалить клиента"))
        self.pushButton_12.setText(_translate("MainWindow", "Добавить счет"))
        self.pushButton_11.setText(_translate("MainWindow", "Редактировать счет"))
        self.pushButton_13.setText(_translate("MainWindow", "Закрыть счет"))
        self.pushButton_14.setText(_translate("MainWindow", "Создать транкзакцию"))
        self.pushButton_16.setText(_translate("MainWindow", "Поиск транзакций"))
        self.pushButton_15.setText(_translate("MainWindow", "Выдать кредит"))
        self.pushButton_17.setText(_translate("MainWindow", "Редактировать кредит"))
        self.pushButton_18.setText(_translate("MainWindow", "Закрыть кредит"))
        self.pushButton_19.setText(_translate("MainWindow", "Добавить карту"))
        self.pushButton_20.setText(_translate("MainWindow", "Привязать карту к счету"))
        self.pushButton_21.setText(_translate("MainWindow", "Удалить карту"))
        self.pushButton_22.setText(_translate("MainWindow", "Добавить сотрудника"))
        self.pushButton_23.setText(_translate("MainWindow", "Редактировать данные о сотруднике"))
        self.pushButton_24.setText(_translate("MainWindow", "Удалить сотрудника"))
        self.pushButton_25.setText(_translate("MainWindow", "Управление ролями"))
        self.pushButton_29.setText(_translate("MainWindow", "Поиск активности"))
        self.pushButton_27.setText(_translate("MainWindow", "Выбрать"))
        self.tabwidget.setTabText(self.tabwidget.indexOf(self.tab), _translate("MainWindow", "Клиенты"))
        self.pushButton_28.setText(_translate("MainWindow", "Выбрать"))
        self.tabwidget.setTabText(self.tabwidget.indexOf(self.tab_2), _translate("MainWindow", "Счета"))
        self.pushButton_30.setText(_translate("MainWindow", "Выбрать"))
        self.tabwidget.setTabText(self.tabwidget.indexOf(self.tab_3), _translate("MainWindow", "Транзакции"))
        self.pushButton_31.setText(_translate("MainWindow", "Выбрать"))
        self.tabwidget.setTabText(self.tabwidget.indexOf(self.tab_7), _translate("MainWindow", "Кредиты"))
        self.pushButton_32.setText(_translate("MainWindow", "Выбрать"))
        self.tabwidget.setTabText(self.tabwidget.indexOf(self.tab_6), _translate("MainWindow", "Карты"))
        self.pushButton_33.setText(_translate("MainWindow", "Выбрать"))
        self.tabwidget.setTabText(self.tabwidget.indexOf(self.tab_5), _translate("MainWindow", "Сотрудники"))
        self.pushButton_34.setText(_translate("MainWindow", "Выбрать"))
        self.tabwidget.setTabText(self.tabwidget.indexOf(self.tab_4), _translate("MainWindow", "Журнал активности"))
        self.pushButton_26.setText(_translate("MainWindow", "Личный кабинет"))
